src/code.less

```diff
@@ -25,4 +25,15 @@ pre {
     border-radius: 3px;
     overflow-x: auto;
     white-space: pre-wrap;
-}
\ No newline at end of file
+}
+pre code{
+    color: inherit;
+    background: transparent;
+    margin-top: -1.50em;
+    display: block;
+    font-size: inherit;
+    padding: initial;
+    white-space:pre-wrap;
+    word-break:break-all;
```

Comment:
Also, for these code snippets. Do we want them to wrap or remain on the same line for horizontal scrolling? It's currently defaulted to wrap 
![image](https://user-images.githubusercontent.com/13633775/33621488-e4f8187a-d9b8-11e7-8deb-46aaab9519b2.png)

but I think it's easier to read on a horizontal scroll. 
![image](https://user-images.githubusercontent.com/13633775/33621524-fe629a74-d9b8-11e7-94db-c2c5a8063a33.png)

As reference, Bootstrap has their code snippets scroll 
![image](https://user-images.githubusercontent.com/13633775/33621576-2ed2f26c-d9b9-11e7-8e70-bbd5975ddbea.png)

Reply:
I agree here. Even Github and Stack overflow follow the same standard for code wrap

Reply:
Nothing inside `pre` tag should wrap.